zhaquirks/tuya/ts0001_fingerbot.py

```diff
@@ -34,7 +34,7 @@ class FingerBotReverse(t.enum8):
     UP_OFF = 0x01
 
 
-class TuyaFingerbotCluster(TuyaMCUCluster):
+class TuyaFingerbotCluster(TuyaEnchantableCluster, TuyaMCUCluster):
```

Comment:
I tried without this and switching the device to use CustomDevice instead of Enchanted, but that did not work. This was the only way to make it work